test/configuration/test_cases/test_36_dns_environment_variable_setup.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DS xmlns="http://www.eprosima.com/XMLSchemas/discovery-server" user_shutdown="false">
+
+  <!-- This test merely assess that the simple participants became full-fledged clients by using the
```

Comment:
My bad :facepalm:. 

```suggestion
  <!-- This test merely assesses that the simple participants become full-fledged clients by using the
```

I think that the `become` verb should be in the present tense.